CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.27.0] - 2025-11-23
```

Comment:
Looks like the link target (to the comparison URL) hasn't been added at the bottom of the file